extensions/opc/include/fetchopc.h

```diff
@@ -91,10 +92,12 @@ class FetchOPCProcessor : public BaseOPCProcessor {
   uint32_t nodesFound_;
   uint32_t variablesFound_;
   uint64_t maxDepth_;
+  bool lazy_mode_;
 
 private:
   std::mutex onTriggerMutex_;
   std::vector<UA_NodeId> translatedNodeIDs_;  // Only used when user provides path, path->nodeid translation is only done once
+  std::map<std::string, std::string> node_timestamp_; // Key = Full path, Value = Timestamp
```

Comment:
I think this would be better as an unordered_map - we don't need ordering and it is potentially very large.